src/lib.rs

```diff
@@ -27,7 +65,7 @@ use warp::Filter;
 ///     .map(|addr: Option<IpAddr>| format!("Hello {}", addr.unwrap()));
 /// ```
 pub fn real_ip(
-    trusted_proxies: Vec<IpAddr>,
+    trusted_proxies: IpNetworks,
 ) -> impl Filter<Extract = (Option<IpAddr>,), Error = Infallible> + Clone {
```

Comment:
I think it's possible to keep the existing API by using the Into trait on the function signature and implementing the appropriate ```impl Into<IpNetworks> for Vec<IpAddr>```. This would avoid needing .into() on the call site.

```suggestion
    trusted_proxies: impl Into<IpNetworks>,
) -> impl Filter<Extract = (Option<IpAddr>,), Error = Infallible> + Clone {
    let trusted_proxies = trusted_proxies.into();
```

Reply:
That sounds the way to go. Thanks for suggestion!